lzhuf.c

```diff
@@ -895,6 +920,22 @@ int Encode (int usock, char *iFile, char *oFile,
    return 0;
 }
 
+static int32 i86ul_to_host(int32 ul)
```

Comment:
I would recommend grouping the two related conversion routines together in the same source file.